ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/EthSignerBaseCommand.java

```diff
@@ -219,6 +219,37 @@ public void setDownstreamHttpPath(final String path) {
       defaultValue = "localhost,127.0.0.1")
   private final AllowListHostsProperty metricsHostAllowList = new AllowListHostsProperty();
 
+  @SuppressWarnings("FieldMayBeFinal") // Because PicoCLI requires Strings to not be final.
+  @Option(
+      names = {"--http-proxy-host"},
```

Comment:
Can you add the `--downstream` prefix to these proxy command options so that they are grouped with the other downstream config.

Reply:
Hello @germinalo, I added a commit to the PR